test/snapshots/markdownlint-test-scenarios.js.md

```diff
@@ -5741,7 +5741,7 @@ Generated by [AVA](https://avajs.dev).
         },
         {
           errorContext: null,
-          errorDetail: 'Expected: # Broken; Actual: # Heading 3 {MD003} {MD043}',
+          errorDetail: 'Expected: # broken; Actual: # heading 3 {md003} {md043}',
```

Comment:
Aha, this looks like a bug. The message here should use the original text, not the text after applying toLowerCase().